Lombiq.Hosting.BuildVersionDisplay/Views/BuildVersion.cshtml

```diff
@@ -0,0 +1,39 @@
+@using System.Reflection;
+@using Lombiq.Hosting.BuildVersionDisplay;
+
+@{
+    var webAppAssembly = Assembly.GetEntryAssembly();
+
+    // These can't be null as long as the code actually compiles.
+#pragma warning disable CS8602 // Dereference of a possibly null reference.
+    var orchardVersion = typeof(OrchardCore.IOrchardHelper).Assembly.GetName().Version.ToString();
+    var buildVersion = webAppAssembly.GetName().Version.ToString();
+#pragma warning restore CS8602 // Dereference of a possibly null reference.
+
+    // When using the module from NuGet, the web app will contain the
```

Comment:
"will contain the" the what? Don't leave me hanging there I want to know the ending. 😄

Reply:
I accidentally the comment :(.